recipes/GKE Setup/hauser.yaml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
I wonder if this would be better to provide as a helm chart and vars file and a terraform module...

@nrmitchi and @bschaeffer may have thoughts.

Reply:
I would agree. A Helm chart here would typically accept a service account to use as a parameter, and the GCP service account would be linked elsewhere. This would allow the basic kubernetes definitions/deployment to be applicable in any kubernetes cluster, and not specific to GKE.

Reply:
I definitely think there's value in adding these, but I wanted to start with a "minimal set" of deps (just gcloud and jq) so that I could get a working GKE recipe quicker. Also, I'm not as familiar with setting up the helm chart (typically in it's own repo?) and I haven't worked with the other clouds enough to know how to write that more generally.

Reply:
But maybe it's simpler than I'm imagining. Just haven't done the research here yet.

Reply:
I don't think a Helm chart that encompasses what you have here would be difficult; it would mostly be wrapping the manifests you've already written, and updating the documentation.

I don't think the lack-of-helm should be considered a blocker here, but we should come back and add it. This is because hauser is safe to remove and recreate, which would make a switch to helm easier for anyone using this.